website/source/docs/agent/index.html.md

```diff
@@ -14,7 +14,7 @@ Vault Agent is a client daemon that provides the following features:
 
 - <tt>[Auto-Auth][autoauth]</tt> - Automatically authenticate to Vault and manage the token renewal process for locally-retrieved dynamic secrets.
 - <tt>[Caching][caching]</tt> - Allows client-side caching of responses containing newly created tokens and responses containing leased secrets generated off of these newly created tokens.
-
+- <tt>[Templating][template]</tt> - Allows rendering of user supplied templates by Vault Agent, using the token generated by the Auto-Auth step. 
```

Comment:
should this match the keyword bellow? `template` vs `templating`

Reply:
The markdown link `template` on line 174 already existed, so I reused it. This list here is a list of features, so I thought "Templating" was the appropriate name. It's a similar difference to `cache` <tt>([cache][caching]` on line 46